core/app/services/spree/locales/set_fallback_locale_for_store.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Locales
+    class SetFallbackLocaleForStore
+      def call(store:)
+        store_default_locale = store.default_locale
+        fallbacks = store.supported_locales_list.each_with_object({}) do |locale, object|
+          object[locale] = [store_default_locale]
+        end
+
+        fallbacks_class = I18n::Locale::Fallbacks
+        fallbacks_instance = fallbacks_class.new(fallbacks)
+
+        Mobility.store_based_fallbacks.value = fallbacks_instance
```

Comment:
Maybe we could abstract this into Mobility in some way? I mean not explicitly set the instance variable on the `Mobility` singleton class. My problem with this now is that it leaks the way we store the fallback in a way. Doing a 
```rb
Mobility.set_store_fallback(fallbacks_instance)
```
would intuitively make more sense to me, since no implementation details are exposed.

Reply:
Yeah, a getter and setter would do the job.

Reply:
Please take a look again